chrome/database.js

```diff
@@ -0,0 +1,181 @@
+
+const database = {
+    requests: [],
+}
+
+// This is crap. The popup and the browser extension have different local/session storages
```

Comment:
Can we briefly explain why do we have this? So that we can change/fix it in the future?

Reply:
Done